lib/DidKeyContainer.ts

```diff
@@ -1,7 +1,3 @@
-/*---------------------------------------------------------------------------------------------
```

Comment:
Thank you

Reply:
the answer is no. This object is no longer used for secrets. I will probably move this out in a next PR. Made the underlying key management simpler 